nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -114,6 +115,11 @@ class LhotseDataLoadingConfig:
     cut_into_windows_hop: Optional[float] = None
     #       III) common options
     keep_excessive_supervisions: bool = True  # when a cut is truncated in the middle of a supervision, should we keep them.
+    #   e. RIR augmentation (synthetic RIR if rir_path is None)
+    #   at the moment supports only Lhotse recording manifests, e.g. https://github.com/lhotse-speech/lhotse/blob/master/lhotse/recipes/rir_noise.py
```

Comment:
Just opening a discussion thread here, would be great to support other alternatives in the future

Reply:
Yes, I'll aim to follow up shortly on this. I'll need to release a new lhotse version that extends the API to support CutSet RIR inputs first, and didn't want to block this work for that.

FWIW constructing a recording set is a one-liner `from lhotse import RecordingSet; RecordingSet.from_dir("my-rirs", pattern="*.wav").to_file("rirs.jsonl.gz")`